arbor/include/arbor/morph/segment_tree.hpp

```diff
@@ -54,6 +54,17 @@ class ARB_ARBOR_API segment_tree {
     friend std::ostream& operator<<(std::ostream&, const segment_tree&);
 };
 
+// Utilities.
+
+// Prune segment tree w.r.t. particular tag
+ARB_ARBOR_API segment_tree prune_tag(const segment_tree& in, int tag);
+
+// Roots of regions of specific tag in segment tree
+ARB_ARBOR_API std::vector<msize_t> prune_tag_roots(const segment_tree& in, int tag);
+
+// Radii at specific distance from tag region roots
```

Comment:
I've removed this function for now to address your above comment on specificity. It's used to calculcate an approximation to Neuron's axon replacement in BluePyOpt. If a default is requested again in the future, this could be reused.